source/mpi_shared_linear_elasticity.cpp

```diff
@@ -446,10 +446,16 @@ namespace Solid
 
       Vector<double> localized_current_displacement(current_displacement);
 
+      for (unsigned int i = 0; i < cellwise_stress.size(); ++i)
+        {
+          cellwise_stress[i].reinit(triangulation.n_active_cells());
+        }
```

Comment:
Do we really need to call ``dealii::Vector<double>::reinit()`` every time we compute the stress? I presume we only need to initialize it once in ``SharedSolidSolver::initialize_system()``.